### DISCUSSION TITLE
Multiple square accounts

### DISCUSSION THREAD

**Discussion by angelrod** [NikolaGavric94/laravel-square#62] [Ideas]
Hi, I hope you are well.

Is there a way to add another square account? I mean I would to like to pass the SQUARE_APPLICATION_ID
and SQUARE_TOKEN manually in order to choose the square's account that I want to use for each transaction that I run.

**Comment by NikolaGavric94** [comment]
Something like this doesn't exist yet, but this might be possible to do in one of the future releases. I'm struggling to see in which case would this be useful tho?
